extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+<!-- omit in toc -->
+# 3DTILES_metadata Extension
+
+**Version 1.0.0**, February 25, 2021
+
+<!-- omit in toc -->
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Sam Suhag, Cesium
+* Patrick Cozzi, Cesium
+* Bao Tran, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against the 3D Tiles 1.0 specification.
+
+Adds new functionality to the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/0.0.0). See [Implicit Tile Metadata](#implicit-tile-metadata).
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is optional, meaning it should be placed in the tileset JSON `extensionsUsed` list, but not in the `extensionsRequired` list.
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+- [Compatibility Notes](#compatibility-notes)
+- [Concepts](#concepts)
+  - [Schemas](#schemas)
+  - [Tileset Metadata](#tileset-metadata)
+  - [Tile Metadata](#tile-metadata)
+    - [Implicit Tile Metadata](#implicit-tile-metadata)
+  - [Group Metadata](#group-metadata)
+  - [Feature Metadata](#feature-metadata)
+  - [Statistics](#statistics)
+  - [Semantics](#semantics)
+  - [Styling](#styling)
+- [JSON Schema Reference](#json-schema-reference)
+
+## Overview
+
+This extension defines a standard mechanism for adding metadata to 3D Tiles. This includes:
+
+* Tileset metadata - metadata about the tileset as a whole.
+* Tile metadata - metadata about individual tiles.
+* Group metadata - metadata about groups of content.
+* Feature metadata - metadata about features. See the companion glTF extension [EXT_feature_metadata](https://github.com/CesiumGS/glTF/tree/master/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0).
+
+<img src="figures/metadata-granularity.png"  alt="Metadata Granularity" width="600">
+
+A tileset defines a **schema**. A schema has a set of **classes** and **enums**. A class contains a set of **properties**, which may be numeric, boolean, string, enum, or array types.
+
+**Entities** (such as tiles, features, etc.) conform to classes and contain **property values**. Depending on the context, property values may be stored in JSON or binary.
```

Comment:
> Depending on the context

What value does this phrase add to the reader?

Probably remove it.

[Make every word tell](https://www.bartleby.com/141/)

Reply:
CC #455 to confirm vocab for "entity", etc.